### ISSUE TITLE
SDL3 VisualC project not building: unresolved external symbols

### ISSUE THREAD

**Issue by AntTheAlchemist** [libsdl-org/SDL_mixer#500]
Windows 10, Win32 build produces these errors:

```
mixer.obj : error LNK2019: unresolved external symbol _Mix_LoadSndFile_RW referenced in function _Mix_LoadWAV_RW
mixer.obj : error LNK2019: unresolved external symbol _SNDFILE_uninit referenced in function _Mix_Quit
```

**Comment by AntTheAlchemist** [comment]
Looks like `load_sndfile.h` and `load_sndfile.c` didn't get added to the project.

**Comment by slouken** [comment]
I'll fix that, thanks.

**Comment by AntTheAlchemist** [comment]
Fixed with a8fd2c1afacf84699f37c1a73f072fcd23560ccb thanks
